web/src/games/bingo/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { IGameState } from './definitions';
+import { MAX_BACKOFF_CANDIDATES } from './constants';
+
+export function shuffleArray(array) {
+  const _array = [...array];
+  // for (let i = _array.length - 1; i > 0; i--) {
+  //   let j = Math.floor(Math.random() * (i + 1));
+  //   let temp = _array[i];
+  //   _array[i] = _array[j];
+  //   _array[j] = temp;
+  // }
```

Comment:
Remove commented out code

Reply:
Opps. this is some DEV thing. This needs to be UNCOMMENTED, otherwise the numbers wont shuffel.